src/authentication/oauth2Auth.ts

```diff
@@ -246,13 +246,25 @@ export class Oauth2Auth extends AlfrescoApiClient {
                 });
             } else {
                 if (this.config.oauth2.silentLogin) {
-                    this.implicitLogin();
+                    if (!this.isInSilentLoginExceptList()) {
+                        this.implicitLogin();
+                    }
                 }
             }
         });
 
     }
 
+    isInSilentLoginExceptList(): Boolean {
```

Comment:
the return type should be boolean with lowercase b

Reply:
`isInSilentLoginExceptList` name sounds like you pass a value to test but we don't do that.